test/sharness/t0250-files-api.sh

```diff
@@ -316,13 +316,26 @@ test_files_api() {
 		verify_dir_contents /cats file1 ipfs this
 	'
 
+	test_expect_success "write 'no-flush' succeeds" '
+		echo "testing" | ipfs files write -f -e /cats/walrus
+	'
+
+	test_expect_success "changes bubbled up to root on inspection" '
+		ipfs files stat / | head -n1 > root_hash
```

Comment:
one thing to note: the `flush=false` option does nothing when running without a daemon. So i'm going to have to special case the test